scrapers/errata.py

```diff
@@ -207,3 +237,44 @@ def get_attached_bugs(self, advisory_id):
         """.format(advisory_id)
         log.info('Getting Bugzilla bugs tied to the advisory with ID {0}'.format(advisory_id))
         return self.teiid.query(sql)
+
+    def get_koji_build(self, build_id):
+        """
+        Query Teiid to find the Koji builds attached to a specific advisory.
+
+        :param int build_id: the build ID
+        :return: a list of a dictionaries
+        :rtype: list
+        """
+        sql = """\
+            SELECT
+                build.extra,
+                package.name as package_name,
+            FROM build
+            LEFT JOIN package ON build.pkg_id = package.id
+            WHERE build.id = {0};
+            """.format(build_id)
+
+        return self.teiid.query(sql)
```

Comment:
Since the method is called `get_koji_build`, this should really return just a single element or `None`, not a list.